shell/plugins/plugin.js

```diff
@@ -28,11 +28,12 @@ export default async(context) => {
   }
 
   if (loadPlugins) {
-    const { store, $plugin } = context;
+    // TODO: Get rancher version using the new API (can't use setting as we have not loading the store)
```

Comment:
if we use some kind of $axios request here it can also be used for `${ UI_PLUGIN_BASE_URL }/index.json`

Reply:
The index.json request need to be authenticated - not sure a raw axios request will work?